static/colors.css

```diff
@@ -0,0 +1,94 @@
+/* --------------------------------------------------------------------------
+ * DARK MODE ---------------------------------------------------------------- */
+
+:root {
+    --bg-primary: black;
+    --bg-secondary: #161616;
+    --bg-code: #333;
+    --bg-mark: rgba(255, 255, 175, 0.8); 
+
+    --text-primary: #aaa;
+    --text-secondary: #6a6a6a; /* #7d7d7d from foot notes */
+    --text-link: #859abc;
+    --text-mark: #222;
+
+    --border-regular: #444; /* thin borders <1px */
+    --border-bold: #353535; /* thicker borders ≥1px */
+
+    --syntax-text: var(--text-primary);
+    --syntax-keyword: #aed7ff;
+    --syntax-entity: #aeafff;
+    --syntax-constant: #d1b0fa;
+    --syntax-string: #d787af;
+    --syntax-symbol: var(--syntax-constant);
+    --syntax-comment: #949494;
+    --syntax-entity-tag: #d7afd7;
+    --syntax-storage-modifier-import: var(--syntax-text);
+
+    --syntax-markup-heading: var(--syntax-text);
+    --syntax-markup-list: var(--syntax-text);
+    --syntax-markup-italic: var(--syntax-text);
+    --syntax-markup-bold: var(--syntax-text);
+    --syntax-markup-inserted-fg: #aff5b4;
+    --syntax-markup-inserted-bg: #033a16;
+    --syntax-markup-deleted-fg: #ffdcd7;
+    --syntax-markup-deleted-bg: #67060c;
+
+    --alert-note: #a5d5fe;
+    --alert-tip: #b4fa72;
+    --alert-important: #ff8ffd;
+    --alert-warning: #ffaf00;
+    --alert-caution: #ff5f5f;
+
+    --ipynb-bg-error: rgba(255, 0, 0, 0.1);
+}
+
+/* --------------------------------------------------------------------------
+ * LIGHT MODE --------------------------------------------------------------- */
+
+@media (prefers-color-scheme: light) {
+    :root {
+        --bg-primary: white;
+        --bg-secondary: #f6f8fa;
+        --bg-code: #afb8c133;
```

Comment:
Btw I think we should change this hex to a simpler form with no transparency info (https://www.quackit.com/css/color/values/css_hex_color_notation_8_digits.cfm)

So:

* `#afb8c133` --> `#eff1f3` (color unchanged)

Agreed? :)

Reply:
Yes I guess this works because there is never anything but white behind it? Fine with me, you can also commit this :)

Reply:
Yeah this is something relevant to GitHub but for us feels like being extra obtuse for no reason :D

Ok I'll do a commit once I decide about the borders. Tbh I think I'm liking the regular=muted for dark as well now that I'm looking at it...

But I'll still double check.